regression/contracts/loop-history-variables-04/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+main.c
+--apply-loop-contracts
+^EXIT=0$
+^SIGNAL=0$
+^\[main.1\] .* Check loop invariant before entry: SUCCESS$
+^\[main.2\] .* Check that loop invariant is preserved: SUCCESS$
+^VERIFICATION SUCCESSFUL$
+--
+--
+This test checks that struct objects are supported for
```

Comment:
1. There are no `struct`s in the test case.
2. We are introducing too many tiny regression test files. Let's just have a single C file with multiple loops, each one checking a different kind of objects. The line number of the errors (whenever that happens) would indicate the root cause; we don't need separate files.